[2025-03-08T13:44:01.570+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.data_to_raw scheduled__2025-03-08T13:43:00+00:00 [queued]>
[2025-03-08T13:44:01.573+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.data_to_raw scheduled__2025-03-08T13:43:00+00:00 [queued]>
[2025-03-08T13:44:01.573+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-08T13:44:01.578+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): data_to_raw> on 2025-03-08 13:43:00+00:00
[2025-03-08T13:44:01.581+0000] {standard_task_runner.py:57} INFO - Started process 2362 to run task
[2025-03-08T13:44:01.583+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'data_to_raw', 'scheduled__2025-03-08T13:43:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmpf55b7sug']
[2025-03-08T13:44:01.585+0000] {standard_task_runner.py:85} INFO - Job 63: Subtask data_to_raw
[2025-03-08T13:44:01.592+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-08T13:44:01.609+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.data_to_raw scheduled__2025-03-08T13:43:00+00:00 [running]> on host cf46f0a3bf1a
[2025-03-08T13:44:01.645+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='data_to_raw' AIRFLOW_CTX_EXECUTION_DATE='2025-03-08T13:43:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-08T13:43:00+00:00'
[2025-03-08T13:44:04.784+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-08T13:44:04.819+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=data_to_raw, execution_date=20250308T134300, start_date=20250308T134401, end_date=20250308T134404
[2025-03-08T13:44:04.883+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-08T13:44:04.915+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
